sys-kernel/coreos-modules/files/commonconfig-4.14

```diff
@@ -931,3 +931,11 @@ CONFIG_MODULE_SIG_KEY="certs/modules.pem"
 # CONFIG_XZ_DEC_ARMTHUMB is not set
 # CONFIG_XZ_DEC_SPARC is not set
 CONFIG_NR_CPUS=512
+CONFIG_ACPI=y
+CONFIG_HW_RANDOM=y
+CONFIG_PCI=y
+CONFIG_PCI_MSI=y
+CONFIG_PCI_IOV=y
+CONFIG_KALLSYMS_ALL=y
+CONFIG_KEXEC=y
```

Comment:
@ezequielgarcia If possible, could you post a diff of the generated kernel config files with and without this change?  That would make it a lot easier to review.

Reply:
Good idea.